### ISSUE TITLE
Very slow use in general for inclusion, config parameters, traffic from 0.30

### ISSUE THREAD

**Issue by koburg** [hassio-addons/addon-zwavejs2mqtt#304]
# Problem/Motivation

> (Why the issue was filed)
Very Slow in general when including new devices, update config parameters on devices
device map is not including all power devices some are grey with no lines.

## Expected behavior
fast inclusion 5-10min this is about the normal time. update config few seconds, map always good 
only grey is battery which is normal 
> (What you expected to happen)

## Actual behavior
very slow when restarting the plugin it takes very long time before devices are updated
when adding new device it just stall for many hours even more than 24 hours normal only 5-10 min
also very slow response on devices when controlling devices. 
> (What actually happened)

## Steps to reproduce
this happen from installing 0.30
> (How can someone else make/see it happen)

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

--- Hardware 
4GB mem , I7 intel processor, VMware running iMac 
usb Z-Stick 7 Aeotec and testet old Stick Gen 5 Aeotec same issue.

**Comment by Lokysek** [comment]
I can confirm very slow reactions and communication from version 0.30 (same result with 0.31) 
For example, Danfoss POPP valves changed set value sometimes after 30mins or never change and show error E5, no connection to GW.
After restore from backup to version 0.29 is all fine as before, values changed while few minutes and no E5 error.

**Comment by koburg** [comment]
I think I have a temporary Fix.

I discovered there was an unusual amount of files in the store folder of the plugin (log was NOT activated).
I backed up the files.

Also my CPU was running like crazy 150%!

**** This step is not recommended unless there is no other way *****
i made a factory reset of the Z-Stick (deleted all devices)
**** This step is not recommended unless there is no other way *****

Deleted all the files in store folder
stopped the plugin.
uninstalled the plugin
reinstall plugin

I did not need to restore files in the store folder.
now the system seems ok.
the Z-Wave is behaving like normal and respond better than before.
the CPU Is just as before this happen about 20%

try these step if you don't have a back up of the older plugin 0.29 .
but don't make factory reset before trying the other steps.
